internal/key/key.go

```diff
@@ -83,3 +88,9 @@ func ReadSecretYamlFromFile(fs afero.Fs, path string) (map[string][]byte, error)
 
 	return rawMap, nil
 }
+
+func OrganizationNamespaceFromName(name string) string {
+	name = fmt.Sprintf(organizationNamespaceFormat, normalize.AsDNSLabelName(name))
```

Comment:
I'd propose to do this other way round:
```suggestion
	name = normalize.AsDNSLabelName(fmt.Sprintf(organizationNamespaceFormat, name))
```

That way the max length trimming of normalization works correctly :slightly_smiling_face:

Reply:
Very good call 👍🏻